Libplanet/Net/Swarm.cs

```diff
@@ -404,6 +404,26 @@ public string TraceTable()
             return Transport is null ? string.Empty : (Transport as NetMQTransport)?.Trace();
         }
 
+        /// <summary>
+        /// Gets the <see cref="PeerChainStatus"/> of the connected <see cref="Peers"/>.
+        /// </summary>
+        /// <param name="dialTimeout">A timeout value for dialing.</param>
+        /// <param name="cancellationToken">
+        /// A cancellation token used to propagate notification that this
+        /// operation should be canceled.
+        /// </param>
+        /// <returns><see cref="PeerChainStatus"/> of the connected <see cref="Peers"/>.</returns>
+        public async Task<IEnumerable<PeerChainStatus>> GetPeerChainStatus(
```

Comment:
The name should end with `Async`.

Reply:
Renamed `GetPeerChainStatus` to `GetPeerChainStatusAsync`. ba45953